mne/commands/tests/test_commands.py

```diff
@@ -276,6 +277,35 @@ def test_flash_bem(tmpdir):
         assert len(tris) == 5120
 
 
+@pytest.mark.slowtest
+@sample.requires_sample_data
+def test_setup_source_space(tmpdir):
+    """Test mne setup_source_space."""
+    check_usage(mne_setup_source_space, force_help=True)
+    # Using the sample dataset
+    subjects_dir = op.join(sample.data_path(download=False), 'subjects')
+    use_fname = op.join(tmpdir, "sources-src.fif")
+    # Test  command
+    with ArgvSetter(('--src', use_fname, '-d', subjects_dir,
+                     '-s', 'sample', '--morph', 'sample', '--verbose')):
+        mne_setup_source_space.run()
```

Comment:
```suggestion
        assert not mne_setup_source_space.run()
```
Also, this will be unnecessarily slow because of the number of points, and distance calculations. Set a small `ico`/`oct` number or a large `spacing` number to minimize the issue.

Reply:
I set testing parameter to ico3.
I'm not sure that `assert not mne_setup_source_space.run()` is what we want. Here, I wanted to test the 'working' case, so this should be `assert not mne_setup_source_space.run() `, but as long as the function returns `None`,  the test does not pass.
So I ended up opening the output file and checking that it's a 2 hemisphere solution.